### PR TITLE
issue #21 patch

### PR DESCRIPTION
**Pull request by ddavlet**

### CODE REVIEW

**Comment by ddavlet** [comment]
https://github.com/ddavlet/webserv/issues/21

**Review by lodeme — DISMISSED** [review]
This does not compile on linux. fixes were pushed to PR#20 and need to be merged into this one here after approval

**Comment by ddavlet** [comment]
@lodeme updated

**Review by olympiaobr — APPROVED** [review]

**Review by lodeme — APPROVED** [review]

**Pull request merged**
